docs/usage/getting-started/running.md

```diff
@@ -93,6 +93,7 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 ### Global config
 
 Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file (`config.js`, `config.json`, `config.json5`, `config.yaml` or `config.yml`), environment variables, or CLI parameters.
```

Comment:
I think that this line above is perhaps misleading. By default we only support `config.js`, but we can now support any file of any name with `.json`, `.json5`, `.yaml` or `.yml` file type

Reply:
Yes, I agree. I updated the paragraph. I believe the any `*.js` named file is also supported with the `RENOVATE_CONFIG_FILE` variable set?

Reply:
Yes